src/babel.js

```diff
@@ -82,13 +82,18 @@ export default function (babel) {
         function buildCallExpression (identifier, tag, path) {
           const built = findAndReplaceAttrs(path, t)
 
-          let { hash, stubs, rules, name } = inline(path.hub.file.code, built, identifierName)
+          let { hash, rules, name } = inline(built, identifierName, 'css')
+
+          // hash will be '0' when no styles are passed so we can just return the original tag
+          if (hash === '0') {
+            return tag
+          }
 
           let arrayValues = parseDynamicValues(rules, t)
 
           const inlineContentExpr = t.functionExpression(
-            t.identifier('inlineCss'),
-            stubs.map((x, i) => t.identifier(`x${i}`)),
+            t.identifier(''),
```

Comment:
Please put this back. If something bombs in here we want to have a proper call stack. Granted it needs a better name like `createEmotionRules`